c2corg_ui/templates/area/edit.html

```diff
@@ -0,0 +1,121 @@
+<%!
+from c2corg_common.attributes import default_langs, quality_types, area_types
+%>
+<%inherit file="../base.html"/>
+<%
+  updating_doc = area_id and area_lang
+%>
+<%namespace file="../helpers/common.html" import="show_title"/>
+<%namespace file="../helpers/edit.html" import="show_preview_container, show_editing_buttons"/>
+
+<%block name="pagetitle">
+<title ng-init="id = ${area_id if area_id else 0}" ng-bind="id ?
+          mainCtrl.page_title('Editing an area') : mainCtrl.page_title('Creating an area')">${show_title('Create/edit area')}
+</title>
+</%block>
+
+<%block name="metarobots"><meta name="robots" content="noindex,nofollow"></%block>
+
+<section class="create-edit-document">
+
+  <h1 ng-init="id = ${area_id if area_id else 0}" class="text-center">
+    <label class="label label-primary" ng-bind="id ? ('Editing an area' | translate) : ('Creating an area' | translate)"></label>
+  </h1>
+
+  <form app-loading app-document-editing="areas" app-document-editing-model="area"
+    app-document-editing-controller-name="appDocumentEditingController"
+  % if updating_doc:
```

Comment:
That would make sense indeed.
But that's not consistent with what we did for other document type editing forms. For instance https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/route/edit.html#L28